src/tactic/norm_num.lean

```diff
@@ -502,6 +502,14 @@ do x₁ ← to_expr x,
 
 end tactic.interactive
 
+namespace tactic
+open norm_num interactive interactive.types
+
+/-- A "non-interactive" wrapper for the full `norm_num` tactic, for use by other tactics. -/
+meta def run_norm_num : tactic unit :=
```

Comment:
Don't we want to give `run_norm_num` the same arguments as `norm_num` for extra flexibility (without the arguments wrapped in the parser monad)?

Reply:
Yes, you're right. I'm going to remove this anyway, as it belongs with the next PR, not here.

(I left off the arguments because my actual use case is resolving the name `run_norm_num` to a `tactic unit` via `eval_expr`, and I was worried about how missing arguments would be handled. I think it's okay; optional arguments will just get filled in during elaboration. In any case, this is for later.)